provider/github/watcher.go

```diff
@@ -195,7 +195,11 @@ func (w *Watcher) processRepoPRs(
 		return c.watchMinInterval, nil
 	}
 
-	if err != nil && !NoErrNotModified.Is(err) {
+	if NoErrNotModified.Is(err) {
+		return c.watchMinInterval, nil
```

Comment:
The previous code was calling `handlePrs` if the error is `NoErrNotModified`, now we skip it. Is it safe in any case? Was it a bug, then?

Reply:
It was not a bug, with this change everything works as before, but I think it's just clearer as we avoid calling `handlePrs`. 
The `doPRListRequest` method returns `nil` for `prs` when a `NoErrNotModified` error occurs, and `handlePrs` just returns a `nil` error if you pass it a zero-length `prs`.

Reply:
I see, thanks